### DISCUSSION TITLE
Segmentation object based on bounding box detection.

### DISCUSSION THREAD

**Discussion by adrianosantospb** [Borda/pyImSegm#26] [Q&A] [answered]
How is it possible use this tools to segmentation an object detected based on bounding box detection (YOLO)? My idea is to get the bboxs detected by the model and do an object segmentation of then.

**Comment by Borda** [comment, marked as answer]
a simple way would be using unsupervised segmentation with GMM and defining two classes... in fact, in such case, it would be very similar to [GrabCut](https://docs.opencv.org/master/d8/d83/tutorial_py_grabcut.html), and for just binary segmentation GrabCut could work better... advantage of this tool is multiclass segmentation over superpixels :]
